### PR TITLE
Scarthgap/rz sbc reserve ECC memory region for RZ/G2L-SBC

### PR DESCRIPTION
**Pull request by nam-vo-khac-rvc**
Hi @preetam-reddy,
This PR include recipes that reserve ECC memory region for RZ/G2L-SBC.
Please help me to check it!

### CODE REVIEW

File: arch/arm/dts/rzpi.dts [hunk 1]
```diff
@@ -31,7 +31,7 @@
 #if (ISSI_4Gb_DDR)
 		reg = <0 0x48000000 0 0x18000000>; //for ISSI DDR 4Gb
 #elif (ISSI_8Gb_DDR)
-		reg = <0 0x48000000 0 0x38000000>; //for ISSI DDR 8Gb
```
Comment (preetam-reddy) [inline arch/arm/dts/rzpi.dts:34]: define values.

Comment (nam-vo-khac-rvc) [reply arch/arm/dts/rzpi.dts:34]: Updated in the commit: 147e0db77c1d0571df468de020ce51836677b4bc

File: arch/arm/dts/rzpi.dts [hunk 2]
```diff
@@ -13,6 +13,20 @@
...
+#define DDR_TOTAL_SIZE    0x40000000  
+#else
+#  error Missing memory side/type define
+#endif
+
+#define DDR_BASE_ADDR    0x48000000
+// 128MB is reserved for secure area and 128MB is reserved for ECC memory 
+#define DDR_RESERVED_SIZE 0x10000000  // 256MB reserved (128MB secure + 128MB ECC)
+#define DDR_AVAILABLE_SIZE (DDR_TOTAL_SIZE - DDR_RESERVED_SIZE)
+
```
Comment (vudangRVC) [inline arch/arm/dts/rzpi.dts]: @nam-vo-khac-rvc ,

Can you move the definition to "include/configs/rzpi.h"? Also, check for duplicated values.

Comment (nam-vo-khac-rvc) [reply arch/arm/dts/rzpi.dts]: Updated in the commit: 147e0db77c1d0571df468de020ce51836677b4bc

**Review by preetam-reddy — APPROVED** [review]

**Pull request merged**